README.md

```diff
@@ -4,13 +4,13 @@
 
 This the awesome book project built with JavaScript
 
-![](file:///home/moise/awsome-books-project/index.html?)
+![](file:///home/moise/awsome-books-project/index.html)
 
 # Awesome books: plain JavaScript with objects
```

Comment:
- [ ] In your README, kindly define instructions on how to get and run the app locally.